src/schemas/models/history.js

```diff
@@ -0,0 +1,55 @@
+const _idSchema = {
+  bsonType: "objectId",
+  type: "string",
+  minLength: 24,
+  maxLength: 24,
+  description: "ID of history"
```

Comment:
```suggestion
  description: "ID of history entry"
```
History là cả lịch sử, và không dùng số nhiều, còn mỗi bản ghi ta có thể gọi là "entry".
Ngoài chỗ này ra xem còn chỗ nào khác thì sửa nốt nhé.